src/components/table-content-wrapper/table-content-wrapper.jsx

```diff
@@ -21,6 +21,7 @@ class TableContentWrapper extends Component {
   render() {
     const {
       classes,
+      captionHidden,
```

Comment:
Wait don't we get both `captionHidden` and `captionVisible` now? Sorry for the inconvenience  😅

Reply:
It should be fine as long as you dont use both

Reply:
but why do we need both then? can't we keep `captionVisible` and just add the object prop possibility?